### PR TITLE
Inclusion of ktlint and dependency file

### PR DESCRIPTION
**Pull request by ezequielsetzesglobant**
The inclusion of ktlint to the project was carried out, in addition the dependencies were refactored into a single file.

View of Project files
image

### CODE REVIEW

**Review by lucianoeli — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

File: di/build.gradle [hunk 1]
```diff
@@ -32,15 +32,22 @@ android {
...
+    implementation Dependencies.core_ktx
+    implementation Dependencies.appcompat
+
+    //UI
+    implementation Dependencies.material
+
+    //Test
+    testImplementation Dependencies.junit
+    androidTestImplementation Dependencies.ext_junit
+    androidTestImplementation Dependencies.expresso_core
```
Comment (nicolbidone) [inline di/build.gradle]: do you need all these dependencies in this module?

File: data/build.gradle [hunk 2]
```diff
@@ -32,14 +32,21 @@ android {
...
+    implementation Dependencies.core_ktx
+    implementation Dependencies.appcompat
+
+    //UI
+    implementation Dependencies.material
+
+    //Test
+    testImplementation Dependencies.junit
+    androidTestImplementation Dependencies.ext_junit
+    androidTestImplementation Dependencies.expresso_core
```
Comment (nicolbidone) [inline data/build.gradle]: do you need all these dependencies in this module?

File: data/build.gradle [hunk 3]
```diff
@@ -32,14 +32,21 @@ android {
     }
 }
 
+preBuild.dependsOn ktlint
```
Comment (nicolbidone) [inline data/build.gradle:35]: if you apply this on every module you can move it to ktlint file

File: di/build.gradle (hunk 1 above)
Comment (ezequielsetzesglobant) [reply di/build.gradle]: done

File: data/build.gradle (hunk 2 above)
Comment (ezequielsetzesglobant) [reply data/build.gradle]: done

File: data/build.gradle [hunk 4]
```diff
@@ -44,9 +44,4 @@ dependencies {
 
     //UI
     implementation Dependencies.material
```
Comment (mauroLonghi) [inline data/build.gradle]: MMM I think that you not need material dependecies in data..

File: di/build.gradle [hunk 5]
```diff
@@ -45,9 +45,4 @@ dependencies {
 
     //UI
     implementation Dependencies.material
```
Comment (mauroLonghi) [inline di/build.gradle]: same here..

Comment (ezequielsetzesglobant) [reply di/build.gradle]: done

File: data/build.gradle (hunk 4 above)
Comment (ezequielsetzesglobant) [reply data/build.gradle]: done

**Review by mauroLonghi — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**